Automattic/woocommerce-payments
# Update readme.txt

P2 post: https://wcpay.wordpress.com/2022/09/30/wp-org-repo-seo-improve-wcpays-plugin-search-ranking/

In line with the work done by @Timstreep, Update wp.org repo listing content for WooCommerce (https://woomarketplace.wordpress.com/2022/09/05/project-proposal-update-wp-org-repo/), we're also introducing a few changes to update the title and tags (WooCommerce Payments Plugin: https://wordpress.org/plugins/woocommerce-payments/) aiming to improve WooCommerce Payments' ranking on the Wordpress.Org Repo search plugins engine.

Fixes #

#### Changes proposed in this Pull Request

**Update WooCommerce Payments listing content**

As of now, the WooCommerce Payments plugin is not well ranked for payment-related keywords on the WordPress.org repo, and the title and tags are two important ranking factors that we can improve. We'll monitor the impact of these changes by using Pugin Rank (https://app.pluginrank.com/).





#### Testing instructions

- Check that the plugin title was changed correctly.
- Check that the plugin tags were changed correctly.

![Alt text](https://cdn-std.droplr.net/files/acc_1233331/PLTjmk "Optional title")

*

-------------------


**Post merge**
